ipatests/test_integration/test_ntp_options.py

```diff
@@ -0,0 +1,226 @@
+        assert expected_msg2 not in client_install.stdout_text
+
+        self.cleanup()
+
+    def test_server_and_client_install_with_multiple_ntp_server(self):
+        """
+        test to verify that ipa-server-install with multiple --ntp-server
+         option
+        """
+        ntp_server1 = "1.pool.ntp.org"
+        ntp_server2 = "2.pool.ntp.org"
+        expected_msg = "Configuration of chrony was changed by installer."
+
+        server_install = tasks.install_master(
+            self.master, setup_dns=False,
+            extra_args=['--ntp-server=%s' % ntp_server1,
+                        '--ntp-server=%s' % ntp_server2])
+        assert expected_msg in server_install.stderr_text
+        cmd = self.master.run_command(['cat', paths.CHRONY_CONF])
+        assert ntp_server1 in cmd.stdout_text
+        assert ntp_server2 in cmd.stdout_text
+
+        client_install = self.install_client('--ntp-server=%s' % ntp_server1,
+                                             '--ntp-server=%s' % ntp_server2)
+        assert expected_msg in client_install.stderr_text
+        cmd = self.client.run_command(['cat', paths.CHRONY_CONF])
+        assert ntp_server1 in cmd.stdout_text
+        assert ntp_server2 in cmd.stdout_text
+
+        self.cleanup()
+
+    def test_server_replica_and_client_install_with_ntp_pool_and_ntp_server(
+            self):
+        """
+        test to verify that ipa-server, ipa-replica and ipa-client install with
+        options --ntp-pool and --ntp-server together
+        """
+        ntp_pool = "pool.ntp.org"
+        ntp_server = "1.pool.ntp.org"
+        expected_msg = "Configuration of chrony was changed by installer."
+
+        server_install = tasks.install_master(
+            self.master, setup_dns=False,
+            extra_args=['--ntp-pool=%s' % ntp_pool,
+                        '--ntp-server=%s' % ntp_server])
+        assert expected_msg in server_install.stderr_text
+        cmd = self.master.run_command(['cat', paths.CHRONY_CONF])
+        assert ntp_pool in cmd.stdout_text
+        assert ntp_server in cmd.stdout_text
+
+        replica_install = self.install_replica('--ntp-pool=%s' % ntp_pool,
+                                               '--ntp-server=%s' % ntp_server)
+        assert expected_msg in replica_install.stderr_text
+        cmd = self.replica.run_command(['cat', paths.CHRONY_CONF])
+        assert ntp_pool in cmd.stdout_text
+        assert ntp_server in cmd.stdout_text
+
+        client_install = self.install_client('--ntp-pool=%s' % ntp_pool,
+                                             '--ntp-server=%s' % ntp_server)
+        assert expected_msg in client_install.stderr_text
+        cmd = self.client.run_command(['cat', paths.CHRONY_CONF])
+        assert ntp_pool in cmd.stdout_text
+        assert ntp_server in cmd.stdout_text
+        tasks.uninstall_master(self.replica)
+
+        self.cleanup()
+
+    def test_server_and_client_install_with_mixed_options(self):
```

Comment:
Please rename method to:
```suggestion
    def test_server_client_install_mixed_options(self):
```